src/customize-pop/gnome-tweaks-extensions/gnome-extensions.md

```diff
@@ -0,0 +1,62 @@
+# GNOME Shell Extensions
+
+GNOME shell extensions are features written by third party developers that build upon the GNOME Shell. Extensions are similar to Chrome Extensions or Firefox Addons.
+
+- **GNOME Extensions** is an application for managing these extensions.
+- **Extension Manager** is a utility for browsing and installing extensions.
+
+Both applications provide menus for configuring, disabling, or removing installed extensions.
+
+---
+
+## Installing the GNOME Extensions & Extension Manager Apps
+
+Launch the Pop!\_Shop and type `extensions` then click `Install` for each application.
+
+![Install Extensions App](/images/gnome-tweaks-extensions/install-extensions-app.png)
+
+## Built-In Extensions
+
+Components packaged in Pop!\_OS are listed in the `Built-In` section. Toggle the switches to enable or disable an extension.
+
+![Built-in Ext](/images/gnome-tweaks-extensions/built-in.png)
+
+These features comprise the Computer Operating System Main Interface Components (COSMIC).
+
+| Extension | Description |
+|-----------|-------------|
+| [Cosmic Dock](https://github.com/pop-os/cosmic-dock) | A modified version of Dash to Dock with different defaults. |
+| [Cosmic Workspaces](https://github.com/pop-os/cosmic-workspaces) | Pop!\_OS's workspaces feature with vertical stacking. |
+| [Pop COSMIC](https://github.com/pop-os/cosmic) | The collection of unique components that comprise the Pop!\_OS experience: Customized Dash-to-Dock, vertical workspaces, and additional configuration options in Settings. |
+| [Pop Shell](https://github.com/pop-os/shell) | A keyboard-driven layer for GNOME Shell which allows for quick and sensible navigation and management of windows. |
+| [System76 Power](https://github.com/pop-os/gnome-shell-extension-system76-power) | A GNOME Shell extension that adds graphical integration to the system76-power daemon. |
+
+## Installing Additional Extensions
+
+Launch the Extension Manager application. Select `Browse` at the top of the window. Use the search bar to find an extension. Click `Install` to install the extension.
+
+![Select Browse](/images/gnome-tweaks-extensions/click-browse.png)
+
+## Changing an Extension's Settings
+
+Developers will include unique settings to tweak the functionality of their extension. Extension settings are managed from the `User-Installed Extensions` list.
```

Comment:
The list is labeled `Manually Installed` in the Extensions app.

The Extensions app is an official GNOME application, while Extension Manager is a third-party application (although upstream GNOME has [accepted support for it](https://gitlab.gnome.org/GNOME/gnome-shell/-/merge_requests/2358).) Was covering settings and removal using Extension Manager instead of the default GNOME Extensions intentional?

Reply:
From my limited experience, it's just so much easier to use the Extension Manager. You can browse and add extensions easily within the app without installing a browser plugin or creating an account. From a privacy and security perspective, I do not like installing browser plugins. I could add the CLI method if you think it's appropriate. I've added this info regarding official support to the application descriptions at the beginning.